### ISSUE TITLE
Firewall does not list ports if no service assigned

### ISSUE THREAD

**Issue by ne20002** [cockpit-project/cockpit#16444]
Cockpit version: 254
OS: Debian Bullseye
Page: Network

I have set up two interfaces, one as 'external', one 'internal'. 

- I changed the port for sshd in /etc/ssh/sshd_config from 22 to XXXX.
- I added port XXXX to firewalld using firewall-cmd

Both, Service ssh and additional Port XXXX (under More Ports) are shown in Cockpit UI for 'external' interface.

- I removed service ssh with Cockpit

For 'external' interface, nothing is shown. firewall-cmd --list-all --zone=external still shows the open port.

 
![grafik](https://user-images.githubusercontent.com/39007279/136577847-1bf125fd-9610-4bcb-b5fc-d33ab95c4d53.png)

**Comment by ne20002** [comment]
Sorry for the late answer.
This happened when I upgraded my VPS to Bullseye and set it up from scratch.
I first installed firewalld from command line, closed everything exept my ssh and wireguard port (both with firewall-cmd --add-port).
I then set up wireguard.
After that I installed cockpit. And it gave the output as shown in first post (which is correct for wg1 and missing the two ports on eth0)
I habe now added a coturn to my VPS and defined the ports for coturn as a Service (via XML). Now ports on external zone are shown correctly (and the Coturn service).
